roles/edpm_nova/tasks/configure.yml

```diff
@@ -12,65 +13,65 @@
     group: "{{ item.group | default(ansible_user) }}"
     mode: "{{ item.mode | default(omit) }}"
   with_items:
-  - { "path": "{{edpm_nova_config_dest}}" }
-  - { "path": "/var/lib/openstack/config/containers"}
-- name: create persistent directories
+    - { "path": "{{ edpm_nova_config_dest }}" }
+    - { "path": "/var/lib/openstack/config/containers" }
```

Comment:
@SeanMooney @gibizer Same as below... which "mode" can we define here?

Reply:
mode is intentionally not defined here but if we were to set it needs to be "0755" to ensure the traversability of the directories. "0750" will break things as will dropping the executable bit and just have read access